src/index.ts

```diff
@@ -276,9 +277,17 @@ export const parsePriceData = (data: Buffer): PriceData => {
   const drv5 = Number(drv5Component) * 10 ** exponent
   const aggregate = parsePriceInfo(data.slice(208, 240), exponent)
 
+  let status = aggregate.status
+
+  if (currentSlot && status === 1) {
+    if(currentSlot - Number(aggregate.publishSlot) > MAX_SLOT_DIFFERENCE) {
+      status = 0
```

Comment:
i think there's an enum you can use for the status values instead of hardcoding the int

Reply:
There is an array. I can use `PirceStatus[status] === "Trading"` but it's not very beautiful though.

I think it's good to change it to a ts type to be able to use it better (in parsing and also checking in different places) but it's out of this PRs scope :D

Reply:
Converted them to enum.